Jenkinsfile

```diff
@@ -6,5 +6,17 @@ node() {
   govuk.setEnvar("PUBLISHING_E2E_TESTS_COMMAND", "test-frontend")
   govuk.buildProject(
     publishingE2ETests: true,
+    extraParameters: [
+      stringParam(
+        name: "GDS_API_ADAPTERS_PACT_VERSION",
+        defaultValue: "master",
+        description: "The branch of gds-api-adapters pact tests to run against"
+      ),
+    ],
+    afterTest : {
+      stage("Verify pact with gds-api-adapters") {
+        govuk.runRakeTask("pact:verify:branch[${env.GDS_API_ADAPTERS_PACT_VERSION}]")
+      }
+    }
```

Comment:
EDIT: we've concluded that this extra Jenkins config is unnecessary, as we're already running the pact tests as part of the default Rake task. This code was removed in https://github.com/alphagov/frontend/pull/2648.